.gitattributes

```diff
@@ -52,20 +52,22 @@ README*                  ocaml-typo=missing-header
 /tools/ci/appveyor/appveyor_build.cmd      ocaml-typo=long-line,very-long-line,missing-header,non-ascii text eol=crlf
 /tools/ci/appveyor/appveyor_build.sh       ocaml-typo=non-ascii
 
+# depend files are auto-generated
+.depend                  ocaml-typo=missing-header
 
 asmcomp/*/emit.mlp       ocaml-typo=tab,long-line,unused-prop
 asmcomp/power/NOTES.md   ocaml-typo=missing-header
 
-/config/gnu              ocaml-typo=prune
+/config/gnu**            ocaml-typo=prune
```

Comment:
This (and the other two changes) are slightly in conflict with the reasoning behind https://github.com/ocaml/ocaml/pull/1148/commits/2adc6af92a60815f76260928ae2829336bf2ade3 - not necessarily a problem. The alternative is to steal the logic from https://github.com/ocaml/ocaml/blob/trunk/tools/ci/travis/travis-ci.sh#L187-L203